kolibri/plugins/device_management/assets/src/state/actions/contentTransferActions.js

```diff
@@ -37,8 +37,10 @@ export function downloadChannelMetadata(store) {
       const { taskId, cancelled } = completedTask;
       if (taskId && !cancelled) {
         return TaskResource.cancelTask(taskId).then(() => {
-          return ChannelResource.getModel(transferredChannel.id).fetch({ file_sizes: true })
-            ._promise;
+          return ChannelResource.getModel(transferredChannel.id).fetch({
+            include_fields:
+              'total_resources,total_file_size,on_device_resources,on_device_file_size',
```

Comment:
I think this should still work if you do it as an array of strings, with the added bonus that it won't look so odd!